wss-agent-api/src/main/java/org/whitesource/agent/api/model/DependencyInfo.java

```diff
@@ -71,23 +68,14 @@
     private boolean deduped;
     private Map<String, String> dependencyModulesToPaths;
     private String euaArtifactId;
-    private Collection<DependencyInfo> aggregatedDependencies;
     private OSInfo osInfo;
 
     /* --- Constructors --- */
 
     /**
      * Default constructor
      */
-    public DependencyInfo() {
-        children = new LinkedList<>();
```

Comment:
It is ok to delete the deleted argument form initialization but why you delete the others? it may affect the app and the ua

Reply:
the other totally removed arguments are not in use anywhere (including in the app) as it was verified as part of this research: https://whitesource.atlassian.net/browse/WSE-5292